fuubar.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = Dir['{test,spec,features}/**/*']
 
   spec.add_dependency             'rspec', ["~> 3.0"]
-  spec.add_dependency             'ruby-progressbar', ["~> 1.4"]
+  spec.add_dependency             'ruby-progressbar', ["~> 1.7"]
```

Comment:
@ojab this change has nothing to do with which `ruby-progressbar` runs during tests.  The `Gemfile.lock` does and it's set to `ruby-progressbar` `1.8`.